Build failed

I have tried on two different ubuntu 20.04 machines and on both I get same errors when building.

`error[E0624]: associated function `new` is private
  --> src/settings.rs:52:27
   |
52 |     let mut con = Config::new();
   |                           ^^^ private associated function
   |
  ::: /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/config-0.12.0/src/config.rs:39:5
   |
39 |     pub(crate) fn new(value: Value) -> Self {
   |     --------------------------------------- private associated function defined here

error[E0061]: this function takes 1 argument but 0 arguments were supplied
  --> src/settings.rs:52:19
   |
52 |     let mut con = Config::new();
   |                   ^^^^^^^^^^^-- supplied 0 arguments
   |                   |
   |                   expected 1 argument
   |
note: associated function defined here
  --> /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/config-0.12.0/src/config.rs:39:19
   |
39 |     pub(crate) fn new(value: Value) -> Self {
   |                   ^^^

warning: use of deprecated associated function `config::Config::merge`: please use 'ConfigBuilder' instead
  --> src/settings.rs:54:9
   |
54 |     con.merge(File::with_name(path.to_str().expect("file name")))?;
   |         ^^^^^
   |
   = note: `#[warn(deprecated)]` on by default

error[E0599]: no method named `try_into` found for struct `config::Config` in the current scope
   --> src/settings.rs:55:9
    |
55  |     con.try_into().map_err(|e| e.into())
    |         ^^^^^^^^ method not found in `config::Config`
    |
    = help: items from traits can only be used if the trait is in scope
    = note: 'std::convert::TryInto' is included in the prelude starting in Edition 2021
help: the following trait is implemented but not in scope; perhaps add a `use` for it:
    |
1   | use std::convert::TryInto;
    |

Some errors have detailed explanations: E0061, E0599, E0624.
For more information about an error, try `rustc --explain E0061`.
warning: `helium-etl-lite` (lib) generated 1 warning
error: could not compile `helium-etl-lite` due to 3 previous errors; 1 warning emitted
warning: build failed, waiting for other jobs to finish...
error: build failed
`

--- a/.gitignore
+++ b/.gitignore
@@ -1,0 +1,3 @@
+/target
+Cargo.lock
+/log

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -14,7 +14,7 @@ path = "src/main.rs"
 helium-jsonrpc = { git = "https://github.com/dewi-alliance/helium-jsonrpc-rs", rev="155b5d2b9a633ae88ed9ee3064b47cf281dba37c" }
 helium-api = { git = "https://github.com/helium/helium-api-rs" }
 tokio = { version = "1.5.0", features = ["full"] }
-config = { version="0", default-features=false, features=["toml"] }
+config = { version= "0.11.0", default-features=false, features=["toml"] }
 serde = "1"
 serde_derive = "1"
 serde_json = "1"

